subtree-splits/blt-project/composer.json

```diff
@@ -0,0 +1,82 @@
+{
+    "name": "acquia/blt-project",
+    "description": "Project template for Drupal 8 sites built with BLT.",
+    "type": "project",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "license": "GPL-2.0-only",
+    "repositories": {
+        "drupal": {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        },
+        "asset-packagist": {
+            "type": "composer",
+            "url": "https://asset-packagist.org"
+        }
+    },
+    "autoload": {
+        "psr-4": {
+            "Acquia\\Blt\\Custom\\": "blt/src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Drupal\\Tests\\PHPUnit\\": "tests/phpunit/src/"
+        }
+    },
+    "config": {
+        "sort-packages": true
+    },
+    "require": {
+        "acquia/blt": "dev-master",
+        "acquia/drupal-spec-tool": "^2.0.0",
+        "acquia/lightning": "^3.1.0",
+        "drupal/acquia_connector": "^1.5.0",
+        "drupal/acquia_purge": "^1.0-beta3",
+        "drupal/cog": "^1.0.0",
+        "drupal/devel": "^1.0.0",
+        "drupal/shield": "^1.2.0",
+        "drupal/qa_accounts": "^1.0.0-alpha1"
+    },
+    "require-dev": {
+        "acquia/blt-require-dev": "dev-master"
```

Comment:
Likewise this should be updated to ^10.0.0 and a release of blt-require-dev should be made.